app/models/claim/pii_scrubber.rb

```diff
@@ -0,0 +1,51 @@
+# Removes personally identifiable information from a claim where the claim is
+# either rejected and hasn't been updated in over two months, or where the claim
+# was scheduled to be paid more than two months ago.
+#
+# Attributes are set to nil, and pii_removed_at is set to the current timestamp.
+
+class Claim
+  class PiiScrubber
+    PII_ATTRIBUTES_TO_DELETE = [
+      :first_name,
+      :middle_name,
+      :surname,
+      :date_of_birth,
+      :address_line_1,
+      :address_line_2,
+      :address_line_3,
+      :address_line_4,
+      :postcode,
+      :payroll_gender,
+      :national_insurance_number,
+      :bank_sort_code,
+      :bank_account_number,
+      :building_society_roll_number,
+    ]
+
+    MINIMUM_DATE_OF_LAST_ACTION = 2.months.ago
+
+    def scrub_completed_claims
+      old_claims_rejected_or_paid.update_all(attribute_values_to_set)
+    end
+
+    private
+
+    def attribute_values_to_set
+      PII_ATTRIBUTES_TO_DELETE.map { |attr| [attr, nil] }.to_h.merge(
+        pii_removed_at: Time.now
```

Comment:
`Time.now` returns the system time. Generally in app code we want to use `Time.zone.now` so that the timestamp accounts for the timezone configured in the application.